packages/jats-xml/src/cli/parse.ts

```diff
@@ -0,0 +1,201 @@
+import { Command } from 'commander';
+import fs from 'fs';
+import { extname } from 'path';
+import { clirun, isUrl, tic, writeFileToFolder } from 'myst-cli-utils';
+import checkdoi from 'doi-utils';
+import chalk from 'chalk';
+import { getSession } from '../session';
+import type { ISession } from '../types';
+import { Tags } from '../types';
+import { Jats } from '../jats';
+import { toText } from 'myst-common';
+import { select, selectAll } from 'unist-util-select';
+import { downloadJatsFromUrl } from '../download';
+import { DEFAULT_RESOLVERS } from '../resolvers';
+import { findDoi, formatDate, toDate } from '../utils';
+
+function hasValidExtension(output: string) {
+  return ['.xml', '.jats'].includes(extname(output).toLowerCase());
+}
+
+async function downloadJats(session: ISession, urlOrDoi: string, output: string): Promise<string> {
+  if (fs.existsSync(urlOrDoi)) {
+    throw new Error(`File "${urlOrDoi}" is local and cannot be downloaded!`);
+  }
+  if (!(checkdoi.validate(urlOrDoi) || isUrl(urlOrDoi))) {
```

Comment:
This check also happens in `downloadJatsFromUrl`, can we delete it here?

Reply:
Yep, done.